packages/asset/deploy/00_set_subscriptions_operator_filter.ts

```diff
@@ -0,0 +1,61 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { DeployFunction } from "hardhat-deploy/types";
+import { DEFAULT_SUBSCRIPTION } from "../constants";
+import { deployments } from "hardhat";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { getNamedAccounts } = hre;
+  const { filterOperatorSubscription } = await getNamedAccounts();
+
+  const operatorFilterRegistry = await deployments.getOrNull(
```

Comment:
Since we should now be able to delete the .json files in `/deployments`, we should not need this section. Please review this file

Reply:
I thought those contracts already exist on mumbai?

Reply:
We can use the real operator filter registry from opensea on mumbai. We should not need deploy scripts for mock files

Reply:
@rishabh0x00 see comment from Maxime